Source/UI/NewUI/PaymentForm/Factory/Theme/BillingForm/Theme+BillingHeader.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Theme+BillingHeader.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 11/08/2022.
+//
+
+import UIKit
+
+public extension Theme {
+    
+    /// Theme generated Billing Header Style
+    struct ThemeBillingHeader: BillingFormHeaderCellStyle {
```

Comment:
```suggestion
    struct BillingHeaderStyle: BillingFormHeaderCellStyle {
```

Reply:
I purposely remove the Style from ending and started objects with Theme. This connects them syntactically to their source (Theme). Also they are only styles from the point of view of the consumers using them to apply styles. As far as creator cares, they are declarations of what they want (ie: no need to add Style to name).

Reply:
To detail further on the connection, you are unable to create Theme objects. They can be discovered via `Theme.Theme{Type}` format but do not expose any initialiser, keeping them to only be created internally by the Theme. This is an intended limitation that encourages good practice (create via design system or go full on DIY).